src/components/Filters/components/Select/styles.ts

```diff
@@ -41,7 +41,7 @@ export const selectStyles: StylesConfig<Option, false> = {
       padding: pxToRem(0, 16),
       background: theme.forms.bgColor,
       border: `${theme.forms.borderWidth} solid ${theme.forms.borderColor}`,
-      borderRadius: theme.borderRadius,
+      borderRadius: theme.radii.default,
```

Comment:
TIL radii is plural of radius 😄

Reply:
yes and `indices` is plural for index but that's coming later 🤣